website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,93 @@
+## Usage
+
+### When to use
+
+- To display and organize tabular data.
+- When comparing, sorting, and filtering multi-dimensional data and objects.
+
+### When not to use
+
+- As a layout mechanism.
+- As a replacement for a spreadsheet or similar application.
+
+### Column placement
+
+<Hds::Table>
+    <:head as |H|>
+        <H.Tr>
+            <H.Th>Start</H.Th>
+            <H.Th>Middle</H.Th>
+            <H.Th>End</H.Th>
+        </H.Tr>
+    </:head>
+</Hds::Table>
+
+- Column placement determines the visual styling based on where the column is placed relatively to other columns in a table header.
+
+!!! Info
+
+The column placement property is only relevant within Figma design tooling and doesn't exist as a property within the production code component.
+
+!!!
+
+### Alignment
+
+<Hds::Table>
+    <:head as |H|>
+        <H.Tr>
+            <H.Th>Start</H.Th>
+            <H.Th>End</H.Th>
+        </H.Tr>
+    </:head>
+</Hds::Table>
+
+#### Alignment Best practices
+
+- Alignment of the header column should remain consistent within the cell (see best practices for alignment within the cell).
+- Columns in the end position frequently use end-alignment when displaying non-string/text content.
+
+### Sorting
+
+![Header column sorting](/assets/components/table/table-header_column-sorting.png)
+
+#### Sorting Best practices
+
+- Sorting is not relevant for all content, therefore the default sort variant of the header column is none.
+  - Whether a column is sortable is left up to the consumer building the table and should be addressed on a case-by-case basis.
```

Comment:
```suggestion
    - Whether a column is sortable is left up to the consumer building the table and should be addressed on a case-by-case basis.
```

Reply:
Although I think this entire line may be able to be removed as the previous line mentions that no sorting is on by default